zwave_js_server/util/lock.py

```diff
@@ -52,7 +57,7 @@ def _get_code_slots(
         slot = {
             ATTR_CODE_SLOT: int(value.property_key),  # type: ignore
             ATTR_NAME: value.metadata.label,
-            ATTR_IN_USE: bool(value.value),
+            ATTR_IN_USE: status_value.value == CodeSlotStatus.ENABLED,
```

Comment:
Are there no tests for this? We should probably tests this since we fixed it here now.

Reply:
There is a test for this -> the fixture I used has three codes set, and both sets of logic worked. My understanding of the implementation on the zwave_js side makes the newer logic *more* correct in that it's like the `value_set` property for OZW

Reply:
Ok. If we can't make it more strict I guess it's ok.